go.sum

```diff
@@ -1,25 +0,0 @@
-github.com/chzyer/logex v1.1.10/go.mod h1:+Ywpsq7O8HXn0nuIou7OrIPyXbp3wmkHB+jjWRnGsAI=
```

Comment:
Probably need to keep `go.sum`

Reply:
I am not sure where the reference to logex came from, but we are not using it currently.

Reply:
Not sure. I'd just run `go mod tidy`

Reply:
yes I think this is OK, running `go mod tidy` just blanks out go.sum